### DISCUSSION TITLE
"a" difference between HDDM and HSSM

### DISCUSSION THREAD

**Discussion by JoeSu112** [lnccbrown/HSSM#282] [Q&A] [answered]
Hi all,

I am trying to do some estimation in HSSM which I have done in the old HDDM package before. And the estimation result is almost the same in all parameters except "a".  And a’s estimation in HSSM is about half of it in HDDM. 

Here is the code in HDDM:

```
m_reg = hddm.HDDMRegressor(subj_data, "v ~ month_delayed + pay_diff", include=('z'),
                  is_group_model=False, keep_regressor_trace=True, p_outlier=0.05)
m_reg.sample(50000, burn=25000)
```

And here is the code in HSSM:
 ```
model_reg = hssm.HSSM(
    data=data,
    model="ddm",
    include=[
        {
           "name": "v",
           "prior": {
                "Intercept": {"name": "Normal", "mu": 0, "sigma": 3},
                "month_delayed": {"name": "Normal", "mu": 0, "sigma": 15},
                "pay_diff": {"name": "Normal", "mu": 0, "sigma": 15},
            },
           "formula": "v ~ 1 + month_delayed + pay_diff",
           "link": "identity",
        },
    ],
    )

    infer_model = model_reg.sample(
        sampler="nuts_numpyro",
        cores=1,  chains=2,
        draws=25000, 
        tune=25000,
        idata_kwargs=dict(log_likelihood=True),
    )
```
I am wondering what makes the difference in "a". It’s because of the prior distribution or definition difference or something else? 

Please let me know what happens, and thanks for any help!

**Comment by AlexanderFengler** [comment, marked as answer]
Hi @JoeSu112 ,

`a_hssm` should be `a_hddm / 2` (that is what you observe right?).

The difference is that HSSM will treat boundaries consistently (intended) as `[-a, a]` whereas in HDDM boundaries were treated as `[0, a]` for basic HDDM functionality and as `[-a, a]` in the HDDMnn classes. 

Sorry about the confusion we should make a note on this in the documentation.
Let me know if any question remains.

Best,
Alex

**Reply by JoeSu112** [reply]
Hi Alex,

Thanks for your quick reply. That was really helpful!!
